db/db.go

```diff
@@ -685,10 +687,13 @@ func (db database) GetAllBounties(r *http.Request) []Bounty {
 			paidQuery = "AND paid = true"
 		}
 	}
+	if orgUuid != "" {
+		orgQuery = "AND org_uuid = " + orgUuid
+	}
 	query := "SELECT * FROM public.bounty WHERE show != false"
 
-	allQuery := query + " " + openQuery + " " + assignedQuery + " " + paidQuery + " " + searchQuery + " " + orderQuery + " " + limitQuery
-
+	allQuery := query + " " + openQuery + " " + assignedQuery + " " + paidQuery + " " + searchQuery + " " + orderQuery + " " + limitQuery + " " + orgQuery
+	fmt.Println(allQuery)
```

Comment:
@wright-eric remove the print line statement

Reply:
I think I have already removed that print line after the latest commit